packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -33,11 +33,13 @@ const ValidDashboardChartTile: FC<{ data: SavedQuery }> = ({ data }) => {
     }, [data]);
 
     return (
-        <SimpleChart
+        <LightdashVisualisation
             isLoading={queryResults.isLoading}
             tableName={data.tableName}
             chartRef={chartRef}
-            chartType={activeVizTab}
+            chartType={
+                activeVizTab as Exclude<DBChartTypes, DBChartTypes.TABLE>
```

Comment:
`LightdashVisualization` accepts any `chartType` it's only `SimpleChart` that accepts `Exclude<DBChartTypes, DBChartTypes.TABLE>`